salt/addons/init.sls

```diff
@@ -14,40 +10,44 @@ include:
     - source:      salt://addons/addons/namespace.yaml.jinja
     - template:    jinja
 
-{% if salt['pillar.get']('addons:dns', 'false').lower() == 'true' %}
-/etc/kubernetes/addons/kubedns-sa.yaml:
-  file.managed:
-    - source:     salt://addons/addons/kubedns-sa.yaml.jinja
-    - template:   jinja
-
-/etc/kubernetes/addons/kubedns-cm.yaml:
-  file.managed:
-    - source:      salt://addons/addons/kubedns-cm.yaml.jinja
-    - template:    jinja
+apply-namespace:
+  cmd.run:
+    - name: |
+        kubectl apply -f /etc/kubernetes/addons/namespace.yaml
```

Comment:
It's definitely an issue, multi master deploys were failing due to kube-dns deployment (after updating them to use apply instead of create), given CI is now multi master, I suspect landing this as is will cause most other changes to hit the issue and fail CI :(